doc/ruleSets/Symfony.rst

```diff
@@ -34,6 +34,7 @@ Rules
   config:
   ``['style' => 'braces']``
 - `empty_loop_condition <./../rules/control_structure/empty_loop_condition.rst>`_
+- `float_literal_case <./../rules/casing/float_literal_case.rst>`_
```

Comment:
I wonder if it shouldn't be one fixer that fixes all numbers

Reply:
☝🏼 exactly my first thought

Reply:
Do you mean for case or for anything related with number format (remove trailing zeros, add `+` after exponent, ..., add literal separator)?

Reply:
I thought about casing, no sure what @keradus meant.

Reply:
I read https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7082#discussion_r1240705494 
as "not having `float_literal_case` and ` integer_literal_case`, but have one `number_literal_case`" which I think is a good idea.

Reply:
for myself: how to deprecate/proxy the original fixer: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7001